src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -269,6 +270,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.xml');
         $loader->load('command.xml');
 
+        if (SvgImagine::class) {
```

Comment:
Are you missing a `class_exists` call or something like that around this? I get an error currently, saying that the `Contao\ImagineSvg\Imagine` class cannot be loaded, if the package is not installed...